ConceirgeDinning.Adapter.USRestaraunt/Models/Result.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+
+namespace ConceirgeDinning.Adapter.USRestaraunt.Models
+{
+    public class Result
```

Comment:
Think a better name for this class.

Reply:
It is according to the contract provided by the supplier